adhocteam/Head-Start-TTADP
# Add react-uswds and react-router

**Description of change**
Add `react-router` and with two routes, `home` and `second page`. These routes will be removed in the future just added them to verify `react-router` is setup properly. Also instead of using `uswds` styles directly we now use the [`react-uswds`](https://github.com/trussworks/react-uswds) library.

**How to test**
 * Pull down and install dependencies (`yarn deps`)
 * Fire up the frontend server (`yarn start`)
 * Browse to `http://localhost:3000`. You should see a navigation menu with two links to two different "pages"

**Issue(s)**
Closes https://github.com/HHS/Head-Start-TTADP/issues/18

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
